src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -182,16 +182,21 @@ protected function isCircularReference($object, &$context)
      * If a circular reference handler is set, it will be called. Otherwise, a
      * {@class CircularReferenceException} will be thrown.
      *
-     * @param object $object
+     * @param object      $object
+     * @param string|null $format
+     * @param array       $context
      *
      * @return mixed
      *
      * @throws CircularReferenceException
      */
-    protected function handleCircularReference($object)
+    protected function handleCircularReference($object/*, ?string $format = null, array $context = array()*/)
```

Comment:
the `?` should be removed
don't miss adding `@final` also as discussed above

Reply:
Maybe, but same as array() vs []: we prefer consistency across branches, and not all of them support this, so it's too early to consider.